docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -83,15 +83,15 @@ When accessing the executor Pod's logs the following error is seen.
 
 *What it means:*
 
-A `NoRouteToHostException` indicates that this Pod is not able to connect to the {WebName} Pod.
+A `NoRouteToHostException` indicates that this pod is not able to connect to the `{WebName}` pod.
 
 *How to resolve it:*
 
-In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the {WebName} Pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
+In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the `{WebName}` pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
```

Comment:
Remove "How to resolve it:" 

This explanation is verbose. Suggested rewrite: "This error message is displayed in a new deployment because the `mta-web-console-executor` pod starts before the `mta-web-console` pod. Wait for the `mta-web-console` pod to start."